include/exadg/utilities/timer_tree.h

```diff
@@ -210,9 +210,39 @@ class TimerTree
   {
     unsigned int const length = get_length();
 
-    pcout << std::endl;
+    unsigned int const max_level = get_max_level();
+
+    if(level <= max_level)
+    {
+      pcout << std::endl;
+
+      do_print_level(pcout, level, 0, length);
+    }
+    else
+    {
+      pcout << std::endl
+            << "Timings can not be printed for level = " << level << "," << std::endl
+            << "since the maximum level of the timer tree is max_level = " << max_level << "."
+            << std::endl;
```

Comment:
I don't see how writing something to pcout is beneficial? Why not assert?

Reply:
In fact I believe it would not be the correct behavior to assert here. The user requests information that can not be provided. But the timing information is nothing vital to the simulation, only "additional information". So the simulation should not crash in my opinion, but terminate "normally" with the message that a particular information can not be given.